ImageAugmentation/imageColorAug.py

```diff
@@ -1,20 +1,34 @@
 from PIL import Image, ImageEnhance
 import os
 
-# hard coding a directory, choose the directory the images are stored in
+# Hard coding a directory, choose the directory the images are stored in
 directory = '/Users/mary/Documents/PersonalProject/images'
-i = 0
+new_file_names = -1
+# Iterates over images that are contained in the directory
 for filename in os.listdir(directory):
-    if filename.endswith(".jpg"):
+    if filename.endswith('.jpg'):
         path = directory + '/' + filename
-        #to save the new image as a separate image
+        # To save the new image as a separate image
         new_path = directory + '/'
         img = Image.open(path)
-        r, g, b = img.split()
 
-        #play around with the combo of bgr to get different colors
-        color = Image.merge('RGB', (b, r, g))
+        # To get the mode of the pic, whether is contains 3 or 4 colors
+        if(img.mode == 'RGB'):
+            # Returns a tuple
+            r, g, b = img.split()   
 
-        #hardcode new path, using {0} with format(i+1) to configure new file name
-        color.save("/Users/mary/Documents/PersonalProject/images/file{0}.jpg".format(i+1))
-    i+=1
+            # Play around with the combo of bgr to get different colors
+            # Images that are made of 3 colors in a channel are refered to as 'RGB'
+            new_img = Image.merge('RGB', (r, g, b))
```

Comment:
Now that these are in the order of r,g,b instead of b,r,g, wouldn't it just return the same colors of the photo, then making it an unaltered photo?